src/types/error.rs

```diff
@@ -36,6 +36,7 @@ impl Object for JsError { }
 
 impl JsError {
     /// Creates a direct instance of the [`Error`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Error) class.
+    #[cfg(feature = "legacy-runtime")]
```

Comment:
#Answer: Since these functions share a function signature, what do you think of putting the cfg flag on the body instead of on the function to ensure conformance?

Reply:
fixed